config/production.json.template

```diff
@@ -1,34 +1,32 @@
 {
   "server": {
-    "port": 3301,
-    "hostname": "http://localhost",
-    "dbKey": "Database 16 Byte Key.",
-    "steamAPIKey": "API Key For Steam Calls.",
-    "sharedSecret": "a secure secret for session sigining.",
-    "clientHome": "http://localhost:8080",
-    "useRedis": true,
-    "apiURL": "http://localhost:8080/api/",
-    "uploadDemos": false
+    "port": $PORT,
```

Comment:
If you're going to change these to envs specifically for Docker, this means project setup as seen in the [wiki](https://github.com/PhlexPlexico/G5API/wiki) would have to be updated. Once again, not opposed to this but I'll need to think some of this over.

Reply:
Using envs seems much better as they can be used with or without docker. Also I think the whole project could be changed to use .env files instead of the config json files. But this is just my opinion :)

Reply:
Honestly this project was started when I was just starting out with node.js development. I'm always down if someone is willing to move it over to .env files, however I know it can be quite a monumental task since these values are kind of littered everywhere in the app. PRs are always welcome, so long as the testing framework finishes successfully! But I would wait until I can get 1.5.0 pushed out as well.